.github/workflows/deploy.yml

```diff
@@ -0,0 +1,29 @@
+name: Deploy
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
Could the regex be more strict, in the (admitedly unlikely) case someone typos a tag?

Reply:
the deploy will fail if the tag doesn't match the tbump regex
```python
    regex = tbump_cfg.version_regex
    version = git_tag[len(prefix) :]
    match = regex.match(version)
    assert match, f"Could not parse {git_tag} as a valid tag"
```